ivy/functional/backends/torch/experimental/layers.py

```diff
@@ -272,6 +272,9 @@ def avg_pool1d(
 
     if data_format == "NWC":
         x = x.permute(0, 2, 1)
+    elif data_format == "NCL":
```

Comment:
why not just use an "or" clause to comebine these two cases as the treatment is the same in both cases right?